submissions/gorodiskiy/HTML-CSS-Popup/index.html

```diff
@@ -138,4 +138,4 @@
   </div>
 </body>
 
-</html>
+</html>
```

Comment:
Don't forget about an empty line at the end of each file. This option can be automated by your code editor check settings.